package.json

```diff
@@ -108,13 +111,16 @@
     "@moderntribe/common": "file:common/src/modules",
     "@moderntribe/tickets": "file:src/modules",
     "@nfen/redux-reducer-injector": "0.0.3",
-    "@wordpress/a11y": "1.0.3",
-    "@wordpress/data": "^1.1.0",
-    "@wordpress/i18n": "^1.2.2",
-    "@wordpress/is-shallow-equal": "^1.1.4",
-    "@wordpress/url": "1.0.0",
+    "@redux-devtools/extension": "^3.3.0",
+    "@the-events-calendar/product-taskmaster": "github:the-events-calendar/product-taskmaster#fix/TEC-5322-wp-67-packages",
```

Comment:
What are we using this for? Zips (moving to pup) and JSON sync (removing).

Is it worth fixing or just removing?

Reply:
In the context of this work, the taskmaster sync is still used and supported.
In reference to my previous comment, this is the PR to make things work, the one to make nice and clean will be the one to move to the use of `@wordpress/scripts`